libs/api/domains/auth/src/lib/auth.service.ts

```diff
@@ -16,4 +19,39 @@ export class AuthService {
   getActorDelegations(user: User): Promise<Array<DelegationDTO>> {
     return this.delegationsApiWithAuth(user).delegationsControllerFindAllTo()
   }
+
+  getAuthScopes(): Promise<any> {
+    // XXX: waiting on auth-public-api implementation
```

Comment:
Not requesting a change. Marking with TODO: makes it visible in the TODO tree in VS Code.

Reply:
Yeah.. both XXX and TODO are highlighted in most IDEs and editors. 
But this will be implemented in a following  PR